google/cloud/forseti/services/inventory/crawler.py

```diff
@@ -25,6 +25,9 @@
 from google.cloud.forseti.services.inventory.base import crawler
 from google.cloud.forseti.services.inventory.base import gcp
 from google.cloud.forseti.services.inventory.base import resources
+from google.cloud.forseti.common.util import log_util
+
+LOGGER = log_util.get_logger(__name__)
```

Comment:
If the logger isn't used, it probably doesn't need to be added.

Reply:
fixed.